modules/sdp/SdpTransformUtil.ts

```diff
@@ -2,12 +2,19 @@ import * as transform from 'sdp-transform';
 
 import { SSRC_GROUP_SEMANTICS } from '../../service/RTC/StandardVideoQualitySettings';
 
+export interface IMLine {
+    direction?: string;
+    ssrcGroups?: Array<{ semantics: string; ssrcs: string; }>;
```

Comment:
Can we define an interface for group and ssrc? I see that these 2 are being used throughout the file.